internals/webpack/webpack.base.babel.js

```diff
@@ -142,6 +142,12 @@ module.exports = options => ({
     .concat(options.plugins)
     .filter(Boolean),
   resolve: {
+    // Setting aliases to make sure that, whenever dependent packages are linked, shared dependencies won't conflict
```

Comment:
You get conflicts when you link another package?
I don't like explicitly naming and aliasing packages here.

Reply:
Getting conflicts when linking `@datapunt/asc-ui`. It conflicts with, at least, Styled components. Setting explicit aliases fixes that problem.